pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -11,15 +11,16 @@ assertExecutable() {
 # makeWrapper EXECUTABLE OUT_PATH ARGS
 
 # ARGS:
-# --argv0       NAME    : set name of executed process to NAME
-#                         (otherwise it’s called …-wrapped)
-# --set         VAR VAL : add VAR with value VAL to the executable’s
-#                         environment
-# --set-default VAR VAL : like --set, but only adds VAR if not already set in
-#                         the environment
-# --unset       VAR     : remove VAR from the environment
-# --run         COMMAND : run command before the executable
-# --add-flags   FLAGS   : add FLAGS to invocation of executable
+# --argv0           NAME    : set name of executed process to NAME
+#                             (otherwise it’s called …-wrapped)
+# --set             VAR VAL : add VAR with value VAL to the executable’s
+#                             environment
+# --set-default     VAR VAL : like --set, but only adds VAR if not already set in
+#                             the environment
+# --unset           VAR     : remove VAR from the environment
+# --run             COMMAND : run command before the executable
+# --add-flags       FLAGS   : add FLAGS to invocation of executable
```

Comment:
Can you also add a note for `--add-flags` that it actually add flags **before** the real arguments?

Reply:
done, PTAL 56c5fd6a061